src/mixins/style-propable.js

```diff
@@ -34,6 +34,7 @@ export default {
   // If you call it twice you'll get a warning anyway.
   prepareStyles() {
     return Styles.prepareStyles.apply(Styles,
-      [(this.state && this.state.muiTheme) || this.context.muiTheme].concat([].slice.apply(arguments)));
+      [(this.state && (this.state._muiTheme || this.state.muiTheme))
+        || this.props._muiTheme].concat([].slice.apply(arguments)));
```

Comment:
@ntgn81 This doesn't matter, soon we'll remove mixins. and this with it. I just left it like this because I wanted this merged faster. in near future I'll remove this mixin so we son't need this method anyway.